### PR TITLE
Fixed view-as-grid and as-list button active state condition

### LINKED ISSUE #8669: view-as-grid and -as-list buttons do not convey currently active state
## Observed behavior

Both buttons are enabled, regardless of which display mode is currently active, resulting in one button doing nothing:

## Expected behavior

The button corresponding to the currently active state should be disabled

## User-facing consequences

The UI is harder to learn and understand because it doesn't convey the "toggle" action visually.

## Context

0.15.0-beta1

### PR DESCRIPTION
**Pull request by akash5100**
## Summary

* Added conditions to disable view-as-grid and view-as-list button 

## References

* closes #8669

## Reviewer guidance

You can check the buttons at learn

screenrecording

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/learn/assets/src/views/LibraryPage.vue [hunk 1]
```diff
@@ -29,13 +29,15 @@
             :color="$themeTokens.text"
             :tooltip="$tr('viewAsList')"
             @click="toggleCardView('list')"
+            :disabled="currentViewStyle === 'list' && true"
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/views/LibraryPage.vue]: what is the purpose of ` && true`?

Comment (akash5100) [reply kolibri/plugins/learn/assets/src/views/LibraryPage.vue:32]: whenever **currentVIewStyle** is a list, it returns a bool (**true**) to `:disabled` 
when the `:disabled` get **true** the button gets disabled.
same for the card view.

Comment (akash5100) [reply kolibri/plugins/learn/assets/src/views/LibraryPage.vue]: Sorry, its useless I will change it

Comment (akash5100) [reply kolibri/plugins/learn/assets/src/views/LibraryPage.vue]: @indirectlylit, I committed the changes and the result is still the same.

**Comment by indirectlylit** [comment]
Thank you @akash5100, looks good!

Lastly, could you please target this change to the `release-0.15.x` branch instead? Then it it will appear in the upcoming release!

**Comment by akash5100** [comment]
@indirectlylit Thanks, I changed the target branch.

**Comment by indirectlylit** [comment]
You'll need to rebase, cherry-pick, or start a new branch from scratch because you started working from `develop` initially:

image

**Comment by akash5100** [comment]
Should I close this PR and open a new one?

**Comment by rtibbles** [comment]
The easiest way to do this is to do:
```
git rebase -i release-v0.15.x
```
to rebase onto the release branch, and then drop all the commits except the ones you have created, then force push to this remote branch.

**Comment by akash5100** [comment]
> The easiest way to do this is to do:
> 
> ```
> git rebase -i release-v0.15.x
> ```
> 
> to rebase onto the release branch, and then drop all the commits except the ones you have created, then force push to this remote branch.

I did `git rebase -i release-v0.15.x` removed all the commits except mine and force pushed to release-v0.15.x

**Pull request closed without merging**